camel/benchmarks/__init__.py

```diff
@@ -12,7 +12,18 @@
 # limitations under the License.
 # ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
 
+# ruff: noqa: E402
```

Comment:
Hi Wendong this is to avoid circular import. By alphabetically order, the `APIBenchBenchmark` and `APIBankBenchmark` are imported before the `BaseBenchmark` is imported. This line is to deliberately delay the imports until `BaseBenchmark` is imported while avoiding autocorrection by ruff.

Reply:
I didn't get ruff error without adding `# ruff: noqa: E402`